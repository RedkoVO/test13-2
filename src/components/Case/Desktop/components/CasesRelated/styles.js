export default () => ({
  root: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'space-between',
    flexWrap: 'wrap',
    backgroundColor: 'rgba(0,0,0,0.13)',
    borderRadius: '20px',
    marginTop: '30px',
    width: '800px',
    padding: '30px 35px',
    boxSizing: 'border-box'
  },
  casesRelatedTilte: {
    flexBasis: '100%',
    textAlign: 'center',
    fontSize: '20px',
    fontWeight: 500,
    marginBottom: '15px'
  },
  casesRelatedItem: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    width: '100px',
    minHeight: '100px',
    backgroundColor: 'rgba(0,0,0,0.13)',
    borderRadius: '15px',
    marginBottom: '10px',
    padding: '5px',
    boxSizing: 'border-box',
    marginRight: 0
  },
  casesRelatedItemImg: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
    width: '95px',
    height: '55px',
    marginBottom: '10px',

    '& img': {
      maxWidth: '100%',
      maxHeight: '100%'
    }
  },
  casesRelatedItemName: {
    fontSize: '10px',
    textAlign: 'center',

    '& span': {
      fontWeight: 700
    }
  }
})
