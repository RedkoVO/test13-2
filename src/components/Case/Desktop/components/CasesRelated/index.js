import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const CasesRelated = ({ classes, relaredItems }) => (
  <div className={classes.root}>
    <div className={classes.casesRelatedTilte}>Related to this case:</div>
    {relaredItems.map(item => (
      <div className={classes.casesRelatedItem} key={item.id}>
        <div className={classes.casesRelatedItemImg}>
          <img src={item.img} alt="" />
        </div>
        <div className={classes.casesRelatedItemName}>
          Gun: <span>{item.title}</span>
        </div>
      </div>
    ))}
  </div>
)

CasesRelated.propTypes = {
  classes: PropTypes.object,
  relaredItems: PropTypes.array
}

export default withStyles(styles)(CasesRelated)
