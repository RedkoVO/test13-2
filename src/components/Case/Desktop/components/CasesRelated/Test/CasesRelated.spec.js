import * as React from 'react'
import renderer from 'react-test-renderer'

import CasesRelated from '../'

import GunCase from '../../../../../../assets/images/caseX.png'

const relaredItems = [
  { id: 1, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 2, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 3, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 4, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 5, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 6, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 7, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 8, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 9, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 10, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 11, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 12, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 13, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 14, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 15, title: 'Good Weapon2 1.2', img: GunCase },
  { id: 16, title: 'Good Weapon2 1.2', img: GunCase }
]

it('renders correctly', () => {
  const tree = renderer
    .create(<CasesRelated relaredItems={relaredItems} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
