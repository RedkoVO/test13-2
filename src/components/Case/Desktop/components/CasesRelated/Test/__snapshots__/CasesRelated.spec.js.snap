// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="CasesRelated-root-1"
>
  <div
    className="CasesRelated-casesRelatedTilte-2"
  >
    Related to this case:
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
  <div
    className="CasesRelated-casesRelatedItem-3"
  >
    <div
      className="CasesRelated-casesRelatedItemImg-4"
    >
      <img
        alt=""
        src="caseX.png"
      />
    </div>
    <div
      className="CasesRelated-casesRelatedItemName-5"
    >
      Gun: 
      <span>
        Good Weapon2 1.2
      </span>
    </div>
  </div>
</div>
`;
